src/AutorestSwift/View Models/ExceptionResponseViewModel.swift

```diff
@@ -52,6 +53,15 @@ struct ExceptionResponseViewModel {
             else { fatalError("Did not find object type for error response") }
         }
 
+        switch objectType {
+        case "String":
+            self.strategy = ResponseBodyType.stringBody.rawValue
+        case "Int":
+            self.strategy = ResponseBodyType.intBody.rawValue
+        default:
+            self.strategy = ResponseBodyType.jsonBody.rawValue
+        }
+
```

Comment:
Exception response can be int or string? Is this a testserver thing?

Reply:
Yes. that is an example in testserver xms-error-responses.json